minecraft-server/start-minecraft.sh

```diff
@@ -187,6 +225,68 @@ if [ ! -e server.properties ]; then
     sed -i "/motd\s*=/ c motd=$MOTD" /data/server.properties
   fi  
 
+  if [ -n "$ALLOW_NETHER" ]; then
```

Comment:
Wow, you knocked out a whole bunch of options here. A helper shell function would be good to avoid the repetition, but I can refactor that later if you don't want to mess with that.